internal/backends/winit/event_loop.rs

```diff
@@ -635,6 +643,17 @@ pub fn run(quit_behavior: i_slint_core::backend::EventLoopQuitBehavior) {
                 winit::event::Event::NewEvents(_) => {
                     corelib::timers::update_timers();
                 }
+
+                winit::event::Event::MainEventsCleared => {
+                    for window in windows_with_pending_property_updates
+                        .drain(..)
+                        .into_iter()
+                        .flat_map(|window_id| window_by_id(window_id))
```

Comment:
why flat_map? map should be fine.

Reply:
`window_by_id` returns an `Option`, so it has to be `flat_map`.

Reply:
i think then the proper term is `filter_map`

Reply:
Ah indeed, that would work, too. It appears that flat_map was added after filter_map? I don’t mind changing to filter_map if you prefer. Just let me know :)